CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## v0.59.0
```

Comment:
* please remove the empty sub sections from 0.59 release (`*Breaking Changes*`, `*experimental*`)
* (optional) please add an "Unreleased" section with empty subsections